docs/modules/ROOT/pages/how-tos/appcat/qppuio-quotas.adoc

```diff
@@ -0,0 +1,35 @@
+= Change AppCat service quota on APPUiO Cloud
+
+It's important, that the service instance has been created with resources lower than the quota.
+This is due to the fact, that the quotas are bound to the namespace and in order to change them, we need the namespace itself.
+
+== Namespace quotas
+The amount of namespaces and thus the amount of AppCat instances that can be created for an organization on APPUiO is by default 25.
```

Comment:
Can you link to external for the number? Should APPUiO Cloud change the value, this documentation will be outdated. Same for every number.